conda-store-server/conda_store_server/server/views/api.py

```diff
@@ -58,11 +87,24 @@ def api_list_environments():
     search = request.args.get("search")
 
     limit, offset = get_paginated_args(request)
+
+    allowed_sorts = {
+        "namespace": orm.Environment.namespace,
```

Comment:
I think this should be `orm.Namespace.name`. Otherwise I believe this will sort by ID's for the namespace.